sunpy/database/database.py

```diff
@@ -627,6 +627,11 @@ def query(self, *query, **kwargs):
 
         return sorted(db_entries, key=operator.attrgetter(sortby))
 
+    @deprecated('0.8', alternative='database.search')
+    def query(self, *query, **kwargs):
+        __doc__ = self.search.__doc__
```

Comment:
I would assume that __doc__ would work for help but I just removed them all so...